pkg/webhook/resources/upgrade/validator.go

```diff
@@ -65,5 +85,58 @@ func (v *upgradeValidator) Create(request *types.Request, newObj runtime.Object)
 		return werror.NewConflict(msg)
 	}
 
+	hasDegradedVolume, err := v.hasDegradedVolume()
```

Comment:
@FrankYang0529 Please also help add the `skipWebhookAnnotation` annotation exception here. 
I think users should have all volumes healthy first before doing an upgrade, but let's still keep a way to skip the check. Thanks!

Reply:
Updated it. Thank you.